stable/mongodb/templates/pvc-standalone.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     chart: {{ template "mongodb.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- with .Values.statefulsetLabels }}
+{{ toYaml . | indent 4 }}
```

Comment:
Wouldn't `{{- toYaml .Values.statefulsetLabels | nindent 4}}` achieve the same as the whole block?

Reply:
@javsalgar Hey, It's nice but it will failed the lint

Reply:
@javsalgar I created the mesh here that's why I close it and reopen it please check https://github.com/helm/charts/pull/20903.